docs/rst/recording/usage/configuration.rst

```diff
@@ -609,6 +600,56 @@ Conversely, setting it to ``warning`` will only show warnings and errors, while
         - ``DDSPIPE`` / ``DDSRECORDER``
         - Regex category or content
 
+By default, the logs will be printed in the standard output.
```

Comment:
Publishing the logs is something completely new, try to introduce the topic more deeply